Makefile

```diff
@@ -1,13 +1,16 @@
 CONTAINER_NAME=packit
-CONTAINER_RUN=podman run --rm -ti -v /tmp/packit:/src:Z $(CONTAINER_NAME)
-TEST_TARGET := ./tests/
+TESTS_INTEGRATION_PATH=tests/integration
+TEST_DATA_PATH=$(TESTS_INTEGRATION_PATH)/test_data
+CONTAINER_RUN=podman run --rm -ti -v $(CURDIR):/src:Z $(CONTAINER_NAME)
```

Comment:
I wonder if we could drop :Z and replace it with `--security-opt label=disable`

Reply:
No idea, I've never used this :-)
I'll try this

Reply:
fixed, seems to work well